api/candlepin-api-spec.yaml

```diff
@@ -9696,3 +9682,33 @@ components:
     basicAuth:
       type: http
       scheme: basic
+
+  parameters:
+    paging_page:
+      name: page
+      description: Page index to return
+      in: query
+      schema:
+        type: integer
+        example: 2
+    paging_per_page:
+      name: per_page
+      description: Number of items to return per page
+      in: query
+      schema:
+        type: integer
+        example: 10
+    paging_order:
+      name: order
+      description: Direction of ordering
+      in: query
+      schema:
+        type: string
+        example: name
+    paging_sort_by:
+      name: sort_by
+      description: Property to use for ordering
+      in: query
+      schema:
+        type: string
+        example: asc
```

Comment:
The two 'example' values for the order and sort_by are inverted here. An example of the `order` param would be 'asc' or 'desc', and an example of the `sort_by` param would be 'name' or 'id'

Reply:
Done.